src/modules/EventList/index.tsx

```diff
@@ -71,15 +74,186 @@ export interface EventListUIProps {
     scrollRef: RefObject<HTMLDivElement>;
     searchTerm?: string;
     setFocus?: Dispatch<SetStateAction<number>>;
+    useConfigOptions: boolean;
 }
 
+interface EventRowProps {
+    refetch?: () => void;
+    event: EventListEvent;
+    maxHits?: number;
+    numMentions: number | null | undefined;
+    index: number;
+    isRefetching: boolean;
+    onSelectEvent?: ChangeHandler<EventListEvent>;
+    renderedRefetch: boolean;
+    searchTerm?: string;
+    setFocus?: Dispatch<SetStateAction<number>>;
+    showDivider: boolean;
+}
+
+const EventRow = ({
```

Comment:
using the EventRow twice... in upcoming events query, and then the recent events query